flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -129,7 +133,7 @@ public boolean isIfNotExists() {
 
 	public void validate() throws SqlValidateException {
 		Set<String> columnNames = new HashSet<>();
-		if (columnList != null) {
+		if (columnList.size() > 0) {
```

Comment:
Correct me if I am wrong but those checks against 0 are useless, no?

Reply:
Yes, they were.

Reply:
@dawidwys  
Yes. I leave them here just for a better code readability and I don't mind removing them here. Do you want to remove them?

Reply:
I think so.

Reply:
I would argue it improves readability. I'd prefer to remove it.

Reply:
+1 to remove it. It is useless.

Reply:
@dawidwys @wuchong Thanks for your suggestion. It's removed now.